libvast/src/command.cpp

```diff
@@ -129,6 +138,23 @@ std::string full_name(const command& cmd) {
   return result;
 }
 
+const command* resolve(const command& cmd,
+                       std::vector<std::string_view>::iterator position,
+                       std::vector<std::string_view>::iterator end) {
+  if (position == end)
+    return &cmd;
+  auto i = std::find_if(cmd.children.begin(), cmd.children.end(),
+                        [&](auto& x) { return x->name == *position; });
+  if(i == cmd.children.end())
+    return nullptr;
+  return resolve(**i, ++position, end);
```

Comment:
It's a nitpick, but `++position` writes to the iterator as a side-effect, although all we need is the next iterator to pass to the function. In short, I think it's better style two write this:

```suggestion
  return resolve(**i, position + 1, end);
```

or

```suggestion
  return resolve(**i, std::next(position), end);
```